cogs/lastactive.py

```diff
@@ -0,0 +1,112 @@
+"""
+lastactive (Ren v2) - Tracks when a user was last active.
+
+How to use:
+    bot.last_active[server_id][channel_id][author_id] -> Retrieves datetime (UTC) of last message sent by that author in
```

Comment:
This looks like it contradicts the way you're storing it below.

Reply:
What do you mean?

Reply:
Fixed in your latest commits, before the `server_id` and `channel_id` were `discord.Server` and `discord.Channel` objects, respectively.